### DISCUSSION TITLE
Is LoRA training via Automatic CLI broken at the moment?

### DISCUSSION THREAD

**Discussion by evshiron** [vladmandic/sdnext#925] [General]
Greetings.

I am currently experimenting LoRA training on RX 7900 XTX, and I examine `cli/train-lora.py` at first because it looks easy to use and it's not using `xformers` and `8bit_adam` which is good for ROCm devices.

However, me with RX 7900 XTX and my friend with RTX 4090 both encounter `AssertionError: duplicated lora name: lora_unet_down_blocks_0_attentions_0_transformer_blocks_0_attn1_to_q` when running `python3 cli/train-lora.py --input dataset/pokemon --output pokemon --tag pokemon`. The dataset comes from [here](https://replicate.com/cloneofsimo/lora-training).

I am a complete newbie to LoRA training and have no idea how it works, and the exception seems to come from https://github.com/kohya-ss/sd-scripts.

Is there anything I am missing? Or is `cli/train-lora.py` just broken for now?

Any idea is appreciated :)

**Comment by vladmandic** [comment]
yes, i'm actually working on it. pretty much everything in `/cli` right now is unreliable as those are my old scripts and changes to core repo in the past few weeks have been massive.

**Reply by evshiron** [reply]
Thank you for your impressive work. I will try other train scripts then.
